site/blocks/ExpandableParagraph.tsx

```diff
@@ -0,0 +1,60 @@
+import classnames from "classnames"
+import React from "react"
+import ReactDOM from "react-dom"
+
+export class ExpandableParagraph extends React.Component<
```

Comment:
This is obviously just personal preference, but I usually prefer very simple React components to be written as [functional components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) (you would then use `useState` for your state).

Many other blocks are also written as functions already, e.g. [TableOfContents](https://github.com/owid/owid-grapher/blob/cd3760757103cc5cd3089a89aea9fc075dc8b79c/site/TableOfContents.tsx#L35).